### ISSUE TITLE
Hotkeys question

### ISSUE THREAD

**Issue by 0Zusu** [minglich/logitech-pubg#18]
So i was reading and editing the code, so heres my question: Could you asing a keyboard key (not Logitech keyboard) to a weapon function?, and how to do it? i tried like "local akm_key= "k" " but it doesnt work that way, maybe is restricted only to the buttons of your mouse?, thx before hand its a very nice code you maked here.

**Comment by minglich** [comment]
you may use AutoHotkey , but i think it's not safe to play pubg .
logitech game soft do not support what you want.

**Comment by 0Zusu** [comment]
yeah AutoHotkey is detected by BES services since a while, but its ok i made a script that it edits  the logitech macro script asigning the same mouse button to a other weapon, like i press the K key and in the code of the logitech script it asings the side mouse button to the AKM clearing first the previous assignation, and so on with other weapons
